topics/python/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-aliases: python3, python2, python-3, python-2, python27, python-3-6, python-3-5, py
+aliases: python3, python2, python-3, python-2, python39, python38, python37, python36, python35, python28, python27, python-39, python-38, python-37, python-36, python-35, python-28, python-27, py
```

Comment:
Many public repositories use the `python-3-6` topic, but none use the `python-36` topic. What's the rationale behind this change?

Reply:
To standardize it, just in case.

Reply:
The community has standardized on the `python-3-6` format, not `python-36`. There are 895 repositories with [the `python-3-6` topic](https://github.com/topics/python-3-6) and zero with [the `python-36` topic](https://github.com/topics/python-36).

It doesn't seem valuable to remove aliases for topics that people are using or add aliases for topics that people are not using, so I'm going to close this PR out. Let me know if I'm misunderstanding, and I'd be open to discussing it further!